src/Servers/IIS/build/repo.targets

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <PropertyGroup>
+    <PackageDependsOn Condition="'$(OS)'=='Windows_NT'">$(PackageDependsOn);PackageNativeProjects</PackageDependsOn>
```

Comment:
I think this entire file can be removed now. It won't be used by the main `build.cmd` run.